service/mantle/shopify/ApiServices.xml

```diff
@@ -395,36 +395,29 @@
         </actions>
     </service>
 
-    <service verb="get" noun="ShopifyCollectionData">
+    <service verb="queue" noun="DeleteCollectionSystemMessage">
         <in-parameters>
             <parameter name="productCategoryId" required="true"/>
         </in-parameters>
-        <out-parameters>
-            <parameter name="shopifyCollectionId"/>
-        </out-parameters>
         <actions>
+            
             <entity-find-one entity-name="mantle.product.category.ProductCategory" value-field="productCategory">
                 <field-map field-name="productCategoryId"/>
             </entity-find-one>
-            <set field="shopifyCollectionId" from="productCategory.pseudoId"/>
-        </actions>
-    </service>
+            <if condition="!productCategory">
+                <message type="warning">The product category ${productCategoryId} does not exist</message>
+                <return/>
+            </if>
 
-    <service verb="queue" noun="DeleteCollectionSystemMessage">
-        <in-parameters>
-            <parameter name="productCategoryId" required="true"/>
-        </in-parameters>
-        <actions>
-            <service-call name="mantle.shopify.ApiServices.get#ShopifyCollectionData"
-                          in-map="[productCategoryId: productCategoryId]"
-                          out-map="context" />
+            <if condition="productCategory.pseudoId ==  productCategory.productCategoryId">
+                <message type="warning">The product category ${productCategoryId} already deleted</message>

```

Comment:
Message: The product category ${productCategoryId} was already deleted.

Reply:
Done